internal/node/node.go

```diff
@@ -130,3 +125,28 @@ func removeLabels(node *v1.Node, labels []string) {
 		)
 	}
 }
+
+func getLabelsToRemove(node *v1.Node, nmcObj *v1beta1.NodeModulesConfig) []string {
+	labelsToRemove := make([]string, 0)
+	for _, module := range nmcObj.Spec.Modules {
+		shouldRemoveLabel := false
+		for _, taint := range node.Spec.Taints {
+			isTolerated := false
+			for _, toleration := range module.Config.Tolerations {
+				if toleration.ToleratesTaint(&taint) {
+					isTolerated = true
+					break
+				}
+			}
+			if !isTolerated {
+				shouldRemoveLabel = true
+				break
+			}
+			shouldRemoveLabel = false
```

Comment:
Why do we need this line? The default value set outside the loop is already `false` isn't it?

Reply:
you are right, I missed this line after my previous fix